README.md

```diff
@@ -1,43 +1,51 @@
-homebrew-neovim
-===============
+# homebrew-neovim
+
 [![Build Status](https://travis-ci.org/neovim/homebrew-neovim.svg?branch=master)](https://travis-ci.org/neovim/homebrew-neovim)
 
-Homebrew formula for neovim.
+Homebrew formula for Neovim.
 
-## Install (stable)
+## Installation
 
-To install the latest stable release:
+Since this formula isn't part of Homebrew, add it as tap first:
 
-    $ brew install neovim/neovim/neovim
+    $ brew tap neovim/neovim
 
-To **upgrade** from a previous version:
+To install the latest stable release:
 
-    $ brew update
-    $ brew upgrade neovim
+    $ brew install neovim
 
-To use Python plugins, see [`:help provider-python`][provider-python].
+To install the latest, potentially unstable, version of Neovim from git HEAD:
 
-## Install (unstable)
+    $ brew install --HEAD neovim
 
-To install the latest "developer version" pulled from the HEAD revision:
+Either of the two options above use the `RelWithDebInfo` build type. This
+enables compiler optimizations and adds debug information to the executable.
 
-    $ brew tap neovim/neovim
-    $ brew install --HEAD neovim
+If you should ever feel the urge to look beneath Neovim's skin, use either of
+the following two lines:
 
-Build the "developer version" with `BUILD_TYPE=release` if you want to disable logging to `~/.nvimlog`:
+    $ brew install --with-dev
+    $ brew install --with-dev --HEAD
 
-    $ brew install --HEAD --with-release neovim
+This uses the `Dev` build type instead. It enables all optimizations that don't
```

Comment:
This uses `BUILD_TYPE=release` build type instead.

Avoids ambiguity and gives cross-references to look for in the faq, CMakeLists.txt, etc.